README.adoc

```diff
@@ -68,3 +68,27 @@ It is in link:.gitignore[.gitignore] so that you can keep your test scenarios un
 == Contributing
 
 Please submit issues and pull requests to the project. You can discuss and ask questions on https://lists.openshift.redhat.com/openshiftmm/listinfo/dev[OpenShift mailing list].
+
+
+=== RVM for dependency management
+
+If you are planning on installing dependencies for verification Tests on your system, it is usually a good idea to use some kind of Version Manager/Package manager, like RVM.
```

Comment:
`usually a good idea`, I'm not sure about it. I just use the OS version of Ruby and rarely, when I need to try another version, then I use RVM. I would change this to `sometimes a good idea`.

Reply:
@akostadinov good point. I, coming from Python world, Ruby Gems installation and management feels very similar to Python Pip package dependencies. In my humble opinion, I still feel its a good idea to have RVM or something similar. I have created a RVM environment named `2.7.1@verification-tests` so that I can use that for this repo while I can create other envs for other repo. So unless someone is planning to have only one ruby project on their system, it feels like a good idea to have separate env. In future if you add another project needing different versions, you will eventually need RVM or something, right? 

I am willing to accept the change you are requesting, just trying to provide my rationale though. Probably you know much more Ruby than I do.

Reply:
I personally use RVM by default instead of messing the system Ruby.  Changing the wording is not that big deal.  Also, @kedark3, you probably put way more information with regards to how to use RVM than needed.  :)

Reply:
@kedark3 , I'm also fine if you use the word "often" instead of "sometimes". I just don't see any reason for most people to install RVM just to mess up with verification-tests.

Another reason I don't like it is because this means installing yet new non-RH built binaries on local workstation which is a potential risk. Not that I'm too much worried.

Reply:
@akostadinov I've sometime seen issue with localhost/vm when doing it against the system ruby.  For example, I recommended @kedark3 to use RVM after his issue with `bundle`  ```bundle install
Ignoring byebug-11.1.3 because its extensions are not built. Try: gem pristine byebug --version 11.1.3
Ignoring json-2.5.1 because its extensions are not built. Try: gem pristine json --version 2.5.1
Ignoring oga-3.3 because its extensions are not built. Try: gem pristine oga --version 3.3
Ignoring psych-3.3.0 because its extensions are not built. Try: gem pristine psych --version 3.3.0
Ignoring racc-1.5.2 because its extensions are not built. Try: gem pristine racc --version 1.5.2
Ignoring ruby-ll-2.1.2 because its extensions are not built. Try: gem pristine ruby-ll --version 2.1.2
Ignoring unf_ext-0.0.7.2 because its extensions are not built. Try: gem pristine unf_ext --version 0.0.7.2
Fetching gem metadata from https://rubygems.org/......
Using concurrent-ruby 1.1.8```

It was straightforward and works when he uses rvm

Reply:
@akostadinov the same applies to @asood-rh I believe.

Reply:
This happens usually after OS upgrade and the native gem extensions need to be rebuilt (or gems removed and reinstalled). If you upgrade your Fedora OS and use RVM, I expect you to hit the same issue.

Reply:
Or maybe it happened because system ruby got updated to a new version. I'm not sure.

Reply:
@akostadinov probably something on my system was not right, but as @pruan-rht pointed out, using RVM was straight forward and easy, keeping separation from system level Ruby. I will update it to "often" as you said in above comment and resolve this thread. Please feel free to re-open if that is not sufficient.